apps/client/src/app/components/home-holdings/home-holdings.component.ts

```diff
@@ -66,8 +66,6 @@ export class HomeHoldingsComponent implements OnDestroy, OnInit {
             this.user.permissions,
             permissions.createOrder
           );
-
-          this.update();
```

Comment:
Without it, holdings are not reloaded when you change the date range.

Reply:
Thanks for this information @dtslvr, I will further dig into this problem and try to resolve it. But I work in a company and this week I will be busy, therefore will try to do it in my free time.

Reply:
Hi @dtslvr, hope you are doing well, I have added a commit, this should hopefully fix the issue, attaching a video for your reference

https://github.com/ghostfolio/ghostfolio/assets/61023134/4d0774be-065a-4c94-83eb-cdfd292a4425

please let me know if there is still any issue....
Thanks 🙂